doc/surf_collide.txt

```diff
@@ -115,7 +116,10 @@ The {specular} style computes a simple specular reflection model.  It
 requires no arguments.  Specular reflection means that a particle
 reflects off a surface element with its incident velocity vector
 reversed with respect to the outward normal of the surface element.
-The particle's speed is unchanged.
+The particle's speed is unchanged. If the optional {adiabatic} keyword
```

Comment:
Specular reflection means that a particle bounce off a surface element
reverses only the component of its velocity normal to the surface.  If the optional {adiabatic} keyword
is used, the particle bounce flips the sign of all 3 xyz components of the particle's incident
velocity, so that it now moves in the opposite direction.  In either case, the particle's speed is unchanged.